pipeline/runtime/runtestsV2.go

```diff
@@ -203,20 +201,21 @@ func getTestsSelection(ctx context.Context, fs filesystem.FileSystem, stepID, wo
 	selection, err = instrumentation.SelectTests(ctx, workspace, filesWithpkg, runOnlySelectedTests, stepID, fs, tiConfig)
 	if err != nil {
 		log.WithError(err).Errorln("An unexpected error occurred during test selection. Running all tests.")
-		return selection, false
+		runOnlySelectedTests = false
```

Comment:
We should make sure, normal test selection is working as we are removing return statement here.